example/cnn_chinese_text_classification/data_helpers.py

```diff
@@ -51,7 +51,7 @@ def get_chinese_text():
         os.system("mkdir data/")
     if (not os.path.exists('data/pos.txt')) or \
        (not os.path.exists('data/neg')):
-        os.system("wget -q https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/example/chinese_text.zip -P data/")
+        os.system("wget -q https://raw.githubusercontent.com/wut0n9/web-data/master/mxnet/example/chinese_text.zip -P data/")
```

Comment:
Is the dmlc one not working for you?

Reply:
Because  the dmlc one hasn't been merge yet,so replace it with my repo .
After it's merged, i can switch dmlc dataset source.